### PR TITLE
feat: Add dark mode support

### PR DESCRIPTION
**Pull request by zbynek**
Adds theme attribute that can be either light (default), dark or auto. That should allow adding dark mode to plugin site before we add it to the others and allow adding a color theme toggle to the page in the future.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -26,7 +26,6 @@ export default {
   input,
   plugins: [
     litcss({
-      include: '/**/*.css',
```
Comment (zbynek) [inline rollup.config.js:29]: It matches css files by default, using regex. It seems that regex works cross-platform and glob is POSIX only.

File: src/jio-locale.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const setLocaleFromUrl = async () => {
   const locale = url.searchParams.get('locale') || (new Intl.NumberFormat()).resolvedOptions().locale || sourceLocale;
   await setLocale(locale);
 };
-setLocaleFromUrl().catch(e => console.error(`Error loading locale: ${(e as Error).message}`));
+setLocaleFromUrl().catch(e => console.log(`Error loading locale: ${(e as Error).message}`));
```
Comment (zbynek) [inline src/jio-locale.ts:24]: Unrelated to dark mode, but not showing this as error on each page load helps with debuging.

**Comment by zbynek** [comment]
@halkeye could you please check/merge this?

File: src/jio-locale.ts (hunk 2 above)
Comment (halkeye) [reply src/jio-locale.ts:24]: Can you make this a seperate fix pr?

**Comment by zbynek** [comment]
> when you change the control it doesn't seem to change anything

It should change the header background from pitch-black (dark mode) to dark gray (light mode); auto mode is identical to one of these, depending on your system theme. If you're using some auto-dark-mode browser feature, the difference may not be visible at all.

**Comment by halkeye** [comment]
Okay I see it now, the colors on my laptop are not that vivid, with the white background it wasn't hugely noticable shift.

Btw, I think https://storybook.js.org/addons/storybook-dark-mode is installed if you want to hook it up to that instead of a custom property, though I think the property override is a good plan.

**Review by halkeye — APPROVED** [review]

**Pull request merged**